classes/person.rb

```diff
@@ -2,14 +2,15 @@
 
 class Person < Nameable
   attr_accessor :name, :age
```

Comment:
- [ ] Same here kindly make sure to add `rentals` to `attr_accessor` so it can be accessible